CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.3] - 2023-09-19
```

Comment:
This is a retro-compatible new feature, so the minor version must be bumped instead of the patch version.

It should be `1.1.0`

Reply:
done